themes/default/content/docs/esc/get-started/retrieve-external-secrets.md

```diff
@@ -0,0 +1,317 @@
+---
+title_tag: Retrieve External Secrets | Pulumi ESC
+title: Retrieve secrets from external sources
+h1: "Pulumi ESC: Retrieve Secrets from External Sources"
+meta_desc: This page provides an overview on how to retrieve secrets from external sources.
+weight: 7
+menu:
+  pulumiesc:
+    parent: esc-get-started
+    identifier: esc-get-started-retrieve-external-secrets
+---
+
+## Overview
+
+With Pulumi ESC, you can store configuration values as plaintext or as secrets directly in the environment file. You also have the option to dynamically import secrets from external sources such as:
```

Comment:
Is there a reference we can make to ensure users know their secret is secure (encrypted, in transit, storage, what level of encryption etc)